webui/react/src/pages/TrialDetails/TrialActions.tsx

```diff
@@ -51,13 +50,9 @@ const TrialActions: React.FC<Props> = ({ trial, trials, onClick, onSettled }: Pr
     setButtonStates(state => ({ ...state, tensorboard: false }));
   }, [ trial.id, onSettled ]);
 
-  const trialCompletedCheckpointSum = useMemo(() => {
-    return trials.reduce((acc, trial) => acc + trial.numCompletedCheckpoints, 0);
```

Comment:
@apizzini I could be missing something here but I think this could run into an issue as this is looking over all trials to find a checkpoint and not just the current trial.

Reply:
Consider that was one of my first tasks and I didn't have a great idea about the code architecture and data model. Might be wrong.

Reply:
I just wanted to check to see if there is something else I'm missing :) I think the change I'm making here should be safe and take care of that.